{{cookiecutter.project_shortname}}/src/lib/fragments/{{cookiecutter.component_name}}.react.js

```diff
@@ -9,33 +9,33 @@ import {defaultProps, propTypes} from '../components/{{cookiecutter.component_na
  * It renders an input with the property `value`
  * which is editable by the user.
  */
-export default class {{cookiecutter.component_name}} extends Component {
-    render() {
-        const {id, label, setProps, value} = this.props;
+const {{cookiecutter.component_name}} = (props) => {
```

Comment:
In this file it should be the conditional to render the template, this is the actual component code for the async components.
```
if component_type == "FunctionComponent":
    FunctionComponent
else:
    ClassComponent
```

Reply:
I’m afraid I’m not following. Can you clarify what needs to be done here? I haven’t changed the code here simply changed it to function component.

Reply:
We just need to do the same we did in the component file but without the `use_async` so the fragments file content should be:

```
{% if cookiecutter.component_type == "Function Component" -%}
    {%- include 'cookiecutter_templates/FunctionComponent.react.js' -%}
{%- else -%}
    {%- include 'cookiecutter_templates/ClassComponent.react.js' -%}
{%- endif -%}
```

Reply:
I changed the content of the fragments file based on your comment.